src/pages/selectSchedule/components/selectPriority/PriorityColumn.tsx

```diff
@@ -11,14 +11,27 @@ function PriorityColumn({ date, timeSlots }: ColumnStructure) {
     ([, slot]) => slot.date === date,
   );
 
+  const changePriorityValue = (priority: number) => {
+    switch (priority) {
+      case 3:
+        return 1;
+      case 2:
+        return 2;
+      case 1:
+        return 3;
+      default:
+        throw Error(`올바르지않은 priority ${priority}`);
```

Comment:
priority가 올바르지 않은 경우는 잘 없겠지만, 만약에 이상한 값이 들어온다면 priority를 0으로 인식하는게 맞다고 생각이 듭니다. priority가 1,2,3인 객체를 제외하면 나머지는 전부 0이어서요! 
그래서 Error를 던지는게 아니라 default는 0으로 반환하면 어떨까요?
```suggestion
      default:
        return 0;
```

Reply:
완전 이해했습니다~ 동의합니다!!